src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -398,36 +437,54 @@ const MarketplaceRow = ({
   }
 
   return (
-    <Row transition="500" marginLeft={selectedMarkets.length > 1 ? '20' : '0'}>
-      {showMarketplaceLogo && (
-        <Column
-          position="relative"
-          cursor="pointer"
-          onMouseEnter={handleHover}
-          onMouseLeave={handleHover}
-          style={{ marginLeft: '-28px' }}
-          onClick={(e) => {
-            e.stopPropagation()
-            removeAssetMarketplace(asset, selectedMarkets[0])
-            removeMarket && removeMarket()
-          }}
-        >
-          <Box className={styles.removeMarketplace} visibility={hovered ? 'visible' : 'hidden'} position="absolute">
-            <Box as="img" width="32" src="/nft/svgs/minusCircle.svg" alt="Remove item" />
-          </Box>
-          <Box
-            as="img"
-            alt={selectedMarkets[0].name}
-            width="28"
-            height="28"
-            borderRadius="4"
-            objectFit="cover"
-            src={selectedMarkets[0].icon}
-            marginRight="16"
-          />
-        </Column>
-      )}
-      <Column style={{ flex: '1.5' }}>
+    <Row transition="500" marginLeft="0">
+      <Column
+        className={bodySmall}
+        color="textSecondary"
+        textAlign="left"
+        flex="1"
+        display={{ sm: 'none', md: 'none', xl: 'flex' }}
+      >
+        {asset.floorPrice ? `${asset.floorPrice.toFixed(2)} ETH` : '-'}
+      </Column>
+      <Column
+        className={bodySmall}
+        color="textSecondary"
+        textAlign="left"
+        flex="1"
+        display={{ sm: 'none', md: 'none', xl: 'flex' }}
+      >
+        {asset.lastPrice ? `${asset.lastPrice.toFixed(2)} ETH` : '-'}
```

Comment:
For most places we go to 3 decimals, can you check with design that 2 is intentional here. For example on Ikegai the floor we show on the collection page is 0.001 but on his list page it shows as 0.00. We also have some auto formatting functions in currency utils that handle trailing zeroes.
![Screen Shot 2022-12-22 at 08 56 38 ](https://user-images.githubusercontent.com/11512321/209188675-43248778-6665-40a9-9262-18c02dd057ff.png)